fixtures/blocks/src/server/Post.block.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+/* eslint-disable import/first */
+
+import * as React from 'react';
+import {block, Suspense} from 'react';
+
+// Server
+
+import loadComments from './Comments.block';
+
+function load(postId) {
+  return {
+    Comments: loadComments(postId),
+  };
+}
+
+// Client
+
+function Post(props, data) {
+  return (
+    <div
+      style={{
+        border: '1px solid #aaa',
+        borderRadius: 10,
+        marginBottom: 20,
+        padding: 20,
+        maxWidth: 500,
+      }}>
+      <h4 style={{marginTop: 0}}>{props.post.title}</h4>
+      <p>{props.post.body}</p>
+      <Suspense
+        fallback={<h5>Loading comments...</h5>}
+        unstable_avoidThisFallback={props.waitForComments}>
```

Comment:
I was planning on turning this into its own component type. So this won't be possible. I also think it indicates an anti-pattern. Like you're trying to control the loading sequence imperatively.

Reply:
What is a good way to encode that I want them to appear with the post in Feed case (to avoid layout jumps), but I want them to load progressively for a Post Permalink case?

Reply:
In that case, I guess you could conditionally render the two different component types and never switch between them.

Maybe add a wrapper to ensure that.

```js
function ResuspendBoundary({ children, fallback }) {
  return <Suspense fallback={fallback} unstable_avoidThisFallback={true}>{children}</Suspense>;
}
```